src/standalone/Selector/MultiSelect.tsx

```diff
@@ -54,6 +54,18 @@ export interface MultiSelectProps<DataT extends MultiSelectorData>
 	subClasses?: {
 		baseSelector: BaseSelectorProps<BaseSelectorData>["classes"];
 	};
+	/**
+	 * Display switch control?
+	 */
+	displaySwitch?: boolean;
```

Comment:
you don't need these, they are inherited from base selector props (see interface extends)

Reply:
> you don't need these, they are inherited from base selector props (see interface extends)
- @TQjona , Updated changes. Please Review.